composer.json

```diff
@@ -19,7 +19,7 @@
         "issues": "https://github.com/humanmade/backupwordpress/issues"
     },
     "require": {
-        "symfony/finder": "~2.6",
-        "ifsnop/mysqldump-php":"2.*"
+        "symfony/finder": "~2.",
+        "ifsnop/mysqldump-php":"~2."
```

Comment:
I was going for the [Next Significant Release syntax](https://getcomposer.org/doc/articles/versions.md#next-significant-release-operators), although looks like I haven't gotten that exactly right. Based on that looks like they should both be ~2.8?